www-src/app/replicator/replicator_backups.coffee

```diff
@@ -160,6 +160,10 @@ module.exports =
 
             toUpload = []
 
+            # Filter images : keep only the ones from Camera
+            # TODO: Android Specific !
+            images = images.filter (path) -> path.indexOf('/DCIM/') != -1
```

Comment:
This depends on android devices vendor and might not work with all phones. However, it may be a good start and we will add other variations when bugs are reported by end users. 
But if we go that way, we need to pop a warning if 0 images match that filter.